#! /bin/sh /usr/share/dpatch/dpatch-run
## 0024_allow_detection_of_heimdal.dpatch by  <roberto@connexer.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: A patch to allow recognition of Heimdal with the heimdal-multidev 
## DP: package

@DPATCH@

Index: trunk/plugins/gssapi.c
===================================================================
--- trunk~/plugins/gssapi.c	(revision 559)
+++ trunk/plugins/gssapi.c	(working copy)
@@ -86,7 +86,7 @@
 
 static const char * GSSAPI_BLANK_STRING = "";
 
-#ifndef HAVE_GSS_C_NT_HOSTBASED_SERVICE
+#if !defined(HAVE_GSS_C_NT_HOSTBASED_SERVICE) && !defined(GSS_C_NT_HOSTBASED_SERVICE)
 extern gss_OID gss_nt_service_name;
 #define GSS_C_NT_HOSTBASED_SERVICE gss_nt_service_name
 #endif
